src/commands/createRepo/GitignoreListStep.ts

```diff
@@ -34,3 +35,8 @@ export class GitignoreListStep extends AzureWizardPromptStep<IStaticWebAppWizard
         return wizardContext.gitignoreExists === false;
     }
 }
+
+
+function shouldIncludeInRepo(pick: QuickPickItem): boolean {
+    return !/^(\.vscode|.*\.log|\.DS_Store)$/.test(pick.label);
```

Comment:
How did you decide on this list of folders/files to ignore?

I think we should use this standard gitignore https://github.com/github/gitignore/blob/master/Node.gitignore to help us decide which files to ignore.

Reply:
Honestly I think I'm leaning to get rid of the prompt and just use that gitignore. There's a lot of things in there that I might not have in my repo _yet_ (so they won't show up in the quick pick), but I probably still want to ignore. SWA is pretty Node focused at the moment anyways, right?

Reply:
I grabbed what looked applicable from this [gist](https://gist.github.com/octocat/9257657).

I see that standard gitignore is for node - should we account for other types of project like Django?

Reply:
Also

![shame](https://i.pinimg.com/originals/03/8a/98/038a98b340fd4d2181e1f820f8060931.gif)

That's for falling into the trap of ignoring the `.vscode` folder, which a lot of VS Code noobs do and causes us problems

Reply:
> I see that standard gitignore is for node - should we account for other types of project like Django?

I didn't see Eric's comment before I posted. I agree it's probs simpler to use the default node gitignore as long as that's the predominant type

Reply:
It's the predominant type, but SWA technically supports Python and C# as well...

Reply:
> Honestly I think I'm leaning to get rid of the prompt and just use that gitignore. There's a lot of things in there that I might not have in my repo _yet_ (so they won't show up in the quick pick), but I probably still want to ignore. SWA is pretty Node focused at the moment anyways, right?

I mean ideally they already have a `.gitignore` in their project so that this is a non-issue.  There's no real way for us to know what their SWA is.  We could try to take a guess based on their app/output locations, but they don't ever select a runtime on creation.

I dunno, I feel like just having everything selected by default isn't the worst experience.  I would imagine most demo repos would already have a `.gitignore`.  A more developed project would probably have a `.gitignore` . And a quick test project doesn't really need the best `.gitignore`.  I'm not totally against having the default node `.gitignore` as a solution but I think it will age poorly.

Reply:
> It's the predominant type, but SWA technically supports Python and C# as well...

I think JS, Python, and C# will be our primary languages for a really long time. If we're not satisfied with always using Node, I think we could come up with a quick pick for the user to pick between those 3. That being said, I think the node gitignore will always be better than a 2-line or empty gitignore, which is what users in the current path would end up with. So I think this change is still an improvement